docker/production/docker-compose.yml

```diff
@@ -14,7 +14,7 @@ services:
       MYSQL_ROOT_PASSWORD: root
   openemr:
     restart: always
-    image: openemr/openemr:7.0.1
+    image: openemr/openemr:7.0.2
```

Comment:
thanks for doing this!
Also need to do this in the docker/production-arm/docker-compose.yml script
(then will bring this PR in)

Reply:
Done.